### DISCUSSION TITLE
Tamaño de un form.

### DISCUSSION THREAD

**Discussion by Bubyana74** [serenity-is/Serenity#5660] [Help]
Buans a Todos!!!!!!!!! 
Gusto en saludarlos. les comento que soy nuevo en el foro y lo que estaría necesitando ya que no lo vi en el listado de temas tratados si alguien sabe y tiene data sobre como customizar el tamaño de un formulario.

Aguado comentarios 

Gracias!!!!

**Comment by VictorTomaili** [comment]
Can you use english if possible?

**Reply by choss21** [reply]
Hola bienvenido, yo igual hablo en español,

lo puedes hacer facilmente con Css in file .less, 
ejemplo

.s-Northwind-ProductDialog {
    > .size { width: 650px; }
    .caption { width: 180px; }
    .s-PropertyGrid .categories { height: 500px; }
}

cualquier duda con gusto te ayudo, 

Saludos
